spec/requests/comments_spec.rb

```diff
@@ -1,7 +1,7 @@
-require 'rails_helper'
+# require 'rails_helper'
 
-RSpec.describe 'Comments', type: :request do
-  describe 'GET /index' do
-    pending "add some examples (or delete) #{__FILE__}"
-  end
-end
+# RSpec.describe 'Comments', type: :request do
+#   describe 'GET /index' do
+#     pending "add some examples (or delete) #{__FILE__}"
+#   end
+# end
```

Comment:
- [ ] The custom method(s) in the **`comment.rb`** file is/are expected to have corresponding unit tests
- [ ] This file should also contain validation tests for comment model.

Reply:
Done!